libraries/AP_SmartAudio/AP_SmartAudio.cpp

```diff
@@ -0,0 +1,947 @@
+#include "AP_SmartAudio.h"
+#include <AP_Math/crc.h>
+#include <GCS_MAVLink/GCS.h>
+
+#define AP_VTX_DEV_MODE     // DEV TESTS ACTIVATE
+#ifdef AP_VTX_DEV_MODE
+# define debug(fmt, args...)	hal.console->printf("\r " fmt "\n", ##args)
```

Comment:
Don't think there is any need for the \r

Reply:
The only way i found to have a readable output using putty is \r i, if not used the lines has the format:
line
      line
            line
                     
Anyway i planned to remove the debug traces and send mavlink messages for final version to notify the important messages.

Reply:
Don't use putty - use mavproxy.py - much easier

Reply:
I started using mavproxy with the mavlink message api to see in console, but in the first debug sessión messages are printed out of order because of the loop freq and the xcess of debugging info. Or there is any other way to deliver the messages with a good throughtput to mavproxy ?

Reply:
Fixed, I didn't know that the mavpproxy also get the serial output directly to de screen.